cgi/import_photos_upload.pl

```diff
@@ -180,10 +180,11 @@
               <span>$Lang{add_photos}{$lang}</span>
               <input type="file" name="files[]" multiple accept="image/*" data-url="/cgi/product_image_import.pl" />
             </span>
+            <!-- automatically start uploads, hide start upload button
```

Comment:
You can just remove the button. If required, it can be restored from git.

Reply:
If we remember that we had it once. :)

Reply:
Removed the commented-out code.